Description: disable test which cannot be run stand-alone
 The 'test_filter' test requires an external source of CT events and
 blocks till these arrive.
Forwarded: not-needed
Author: Jeremy Sowden <jeremy@azazel.net>
Last-Update: 2022-04-09

--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -5,7 +5,7 @@
 
 EXTRA_DIST = qa-connlabel.conf
 
-TESTS = test_api test_filter test_connlabel
+TESTS = test_api test_connlabel
 
 test_api_SOURCES = test_api.c
 test_api_LDADD = ../src/libnetfilter_conntrack.la
